Library/Formula/rgxg.rb

```diff
@@ -0,0 +1,15 @@
+class Rgxg < Formula
+  desc "rgxg (ReGular eXpression Generator)"
```

Comment:
Don't need to use the formula name inside the `desc`. It'll be automatically prepended when people see it.